config/runtime.exs

```diff
@@ -4,7 +4,12 @@
 # remember to add this file to your .gitignore.
 import Config
 
-if config_env() != :test do
+if config_env() == :test do
+  # always force locale to en for tests:
+  default_locale = "en"
+  config :gettext, :default_locale, default_locale
+  config :timex, :default_locale, default_locale
```

Comment:
I think, you can do the config part after the if block, in order to save 2 lines of code ;) you yould either return the language in the if statement or declare the variable before.

Just an example with case:
```
result = 
  case {a,b} do
    _ when a > b -> "a"
    _ when a < b -> "b"
    _ -> "equal"
  end
```

Reply:
Yes, I thought this would be confusing, because setting the language would happen way down the file. I can switch if you prefer it that way.